backend/server/mailing.go

```diff
@@ -2,91 +2,239 @@ package main
-func FeedbackEnquiry(targetEmail string) echo.HandlerFunc {
+// HandleFeedback - validates feedback input and forwards as an email to the revelant inbox
+func HandleFeedback() echo.HandlerFunc {
 	return func(c echo.Context) error {
-		name := c.FormValue("name")
-		email := c.FormValue("email")
-		body := c.FormValue("body")
-
-		// if there is no email, use the struct where emails are not required (FeedbackNoEmail)
-		if email == "" {
-			feedback := FeedbackNoEmail{
-				Name:  name,
-				Email: email,
-				Body:  body,
-			}
+		// Extract fields from form
+		feedback := Feedback{
+			Name:  c.FormValue("name"),
+			Email: c.FormValue("email"),
+			Body:  c.FormValue("body"),
+		}
 
-			if err := c.Validate(feedback); err != nil {
-				return c.JSON(http.StatusBadRequest, H{
-					"error": err,
-				})
-			}
+		// Validate struct
+		if err := c.Validate(feedback); err != nil {
+			return c.JSON(http.StatusBadRequest, H{
+				"error": err,
+			})
+		}
 
-			// if there is an email, check that the email is valid by using the Feedback Struct
-		} else {
-			feedback := Feedback{
-				Name:  c.FormValue("name"),
-				Email: c.FormValue("email"),
-				Body:  c.FormValue("body"),
-			}
+		// Add to bundle
+		feedbackBundle = append(feedbackBundle, feedback)
 
-			if err := c.Validate(feedback); err != nil {
-				return c.JSON(http.StatusBadRequest, H{
-					"error": err,
-				})
-			}
+		return c.JSON(http.StatusAccepted, H{})
+	}
+}
 
-		}
+/////////
+// TIMERS
+/////////
+
+// This function is executed once in a subroutine and triggers every 15 minutes
+func enquiryMailingTimer() {
+	for {
+		time.Sleep(15 * time.Minute)
+		DispatchEnquiryBundles()
+	}
+}
+
+// This function is executed once in a subroutine and triggers once every day
+func feedbackMailingTimer() {
+	for {
+		time.Sleep(24 * time.Hour)
+		DispatchFeedbackBundle()
+	}
+}
 
-		//TODO: Compose email and send
-		return c.JSON(http.StatusOK, H{})
+//////////////
+// DISPATCHERS
+//////////////
+
+// DispatchEnquiryBundles - public trigger for dispatching enquiries
+func DispatchEnquiryBundles() {
+	go sendEnquiryBundle(infoEmail, &infoBundle)
+	go sendEnquiryBundle(sponsorshipEmail, &sponsorshipBundle)
+}
+
+// DispatchFeedbackBundle - public trigger for dispatching feedbacks
+func DispatchFeedbackBundle() {
+	go sendFeedbackBundle(infoEmail, &feedbackBundle)
+}
+
+/////////////////
+// BUNDLE SENDERS
+/////////////////
+
+func sendEnquiryBundle(targetEmail string, bundle *[]Enquiry) {
+	if len(*bundle) == 0 {
+		return
+	}
+
+	// Format message payload
+	payload := []mailjet.InfoMessagesV31{
+		mailjet.InfoMessagesV31{
+			From: &mailjet.RecipientV31{
+				Email: "projects.website@csesoc.org.au",
+				Name:  "CSESoc Website",
+			},
+			To: &mailjet.RecipientsV31{
+				mailjet.RecipientV31{
+					Email: targetEmail,
+				},
+			},
+			Subject:  "Website enquiry bundle",
+			HTMLPart: joinEnquiries(*bundle),
```

Comment:
Does this mean each information such as enquiries would be bunched together as part of the description? Does it look confusing if there are alot of different emails happening? @glebme lmk what you think

Reply:
I actually formatted with HTML so it looks pretty readable (I might be biased since I wrote it though 🤣). I attached a screenshot of the formatting as a comment to the [Jira ticket](https://compclub.atlassian.net/browse/CW-44) corresponding to this PR. Let me know what you guys think.